City-of-Helsinki/drupal-helfi-form-tool
# LOM-276: Logout reminder/confirmation modal

# [LOM-276](https://helsinkisolutionoffice.atlassian.net/browse/LOM-276)


## What was done

* Modal dialog to confirm moving out of the service
## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/LOM-276-logout-reminder`
  * `make fresh`
* Run `make drush-cr`
* Style build

## How to test


* [ ] Check that external links fire confirmation dialog (example frontpage links & form submitted succesfully page "Close and logout" button
* [ ] Check that translations are set correctly
* [ ] Accessability (Tab focus & will focus the original element when closed)

`confirm-dialog` css-class can be used to force this dialog to appear (in-case we need to fine-tune external links or allowlist some links to skip the dialog)


[LOM-276]: https://helsinkisolutionoffice.atlassian.net/browse/LOM-276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
